employer_recommendation_system/mailer/models.py

```diff
@@ -1,3 +1,48 @@
 from django.db import models
 
 # Create your models here.
+
+
+class EmailRecord(models.Model):
+    """
+    Stores individual email addresses and their sending status.
+    Each row corresponds to one email address from the uploaded CSV file.
+    """
+    email_address = models.EmailField(unique=True)
```

Comment:
Consider adding for faster lookups by email.:

class Meta:
    indexes = [models.Index(fields=['email_address'])]